kivy/uix/textinput.py

```diff
@@ -1402,7 +1402,8 @@ def on_touch_up(self, touch):
             # show Bubble
             win = EventLoop.window
             if self._selection_to != self._selection_from:
-                self._show_cut_copy_paste(touch.pos, win)
+                touch_rel_pos = (touch.pos[0] - self.x, touch.pos[1] - self.y)
```

Comment:
The show_cut_copy_paste function is supposed to manage the display of the bubble,
 thus it should be taking care of the conversion if any, this has the added advantage that we won't have the conversion in many places.